dataminer-overview/Training_courses_and_certification/Overview_Training_certification.md

```diff
@@ -16,79 +16,99 @@ For example, the DataMiner Fundamentals Certification is intended for DataMiner
 > For more information about the different certifications and their process, please visit [DataMiner Certification](https://community.dataminer.services/learning/certification/).
 
 ## DataMiner Fundamentals exam guide
-This section is designed to help you understand the process, exam format, scoring criteria, and provide you with preparation tips. Our goal is to ensure you have all the information you need to succeed. 
 
-### Certification process 
-As soon as the ordering process is finalized and your certification process can be started, you receive an automated email from Skyline Academy with information about the process and the deadline. 
+Are you preparing to take the DataMiner Fundamentals exam? Below you can find all the information you need, including the certification process, the exam format, the scoring criteria, and tips on how to prepare for the exam.
+
+### Certification process
+
+As soon as the ordering process is finalized and your certification process can be started, you will receive an automated email from Skyline Academy with information about the process and the deadline.
+
+You will also receive an email with your user ID and password to access the training DMA in order to practice and to take the exam.
+
+To book an appointment, you will need to use the booking link that you will find in the email you received with information about the start of the certification process.
```

Comment:
I also want to add the info that candidates receive monthly reminders by email so don't forget about the process. 
20, 10, 5 days before the deadline they also receive emails to inform them that their process will expire soon. Once expired they also receive an email. 
This information is the same for the configurator process